packages/debug-utils/index.js

```diff
@@ -39,8 +39,8 @@ const commandReference = {
   "u": "step out",
   "n": "step next",
   ";": "step instruction (include number to step multiple)",
-  "p": "print instruction & state (`p [mem|cal|sto]*`; see docs for more)",
-  "l": "print additional source context",
+  "p": "print instruction & state (`p [mem|cal|sto|line-number]*`; see docs for more)",
+  "l": "print additional source context (`l [line-number]`; see docs for more)",
```

Comment:
Thanks for updating this!  I don't think this quite conveys what we want, though.  Firstly, this doesn't include the necessary `+/-`.  Secondly, "line number" means the ordinal number of a given line of code, not a cardinal count of lines of count.  How about `l [+<lines-ahead>] [-<lines-back>]`?

Reply:
Oops, previous comment got a bit garbled in places, fixed it now.